sklearn/ensemble/_stacking.py

```diff
@@ -332,11 +337,22 @@ class StackingClassifier(ClassifierMixin, _BaseStacking):
 
         * if 'auto', it will try to invoke, for each estimator,
           `'predict_proba'`, `'decision_function'` or `'predict'` in that
-          order.
+          order;
         * otherwise, one of `'predict_proba'`, `'decision_function'` or
          `'predict'`. If the method is not implemented by the estimator, it
          will raise an error.
 
+    drop : 'first', int, str or None, default='first'
```

Comment:
'first' is now ambiguous if 'first' is the name of one of the classes! Do we need to require that `'first' not in classes_ or classes_[0] == 'first'`?!!

Reply:
Rather, let's use {True, False, str}